apps/ledger-live-desktop/src/renderer/components/ErrorBanner.tsx

```diff
@@ -2,20 +2,26 @@ import React from "react";
 import { urls } from "~/config/urls";
 import TranslatedError from "./TranslatedError";
 import Alert from "./Alert";
+
 type Props = {
   error: Error;
   warning?: boolean;
+  fallback?: {
```

Comment:
This is needed to provide the `fallback` prop to the underlying `TranslatedError` component.
As we can see in `TranslatedError`, if the provided error does not have a key for the specified field (either no `title` or no `description`) AND does not have a `fallback` prop provided, the generic error is used.

cf.
https://github.com/LedgerHQ/ledger-live/blob/47ea0d6fc91048555eea0db1f8ddab08f1fcac8c/apps/ledger-live-desktop/src/renderer/components/TranslatedError/TranslatedError.tsx#L71-L72

And since some errors, like the `NotEnoughGas` one, don't have description, we use an empty fallback in order not to display the generic `Something went wrong. Please retry or contact Ledger Support.` error in this case

https://github.com/LedgerHQ/ledger-live/blob/d5dae1e738f76dcd030b45d7723e503ad89ba311/apps/ledger-live-desktop/src/renderer/families/evm/EditTransaction/components/TransactionErrorBanner.tsx#L5-L29